shared/chat/conversation/input.desktop.js

```diff
@@ -52,12 +54,11 @@ class Conversation extends Component<void, Props, State> {
       return
     }
 
-    // Allow copy
-    if (ev.metaKey && ['Meta', 'c'].includes(ev.key)) {
-      return
+    const isPasteKey = ev.key === 'v' && (ev.ctrlKey || ev.metaKey)
```

Comment:
That's a good idea! I think it might be possible, but it may require us to insert the text manually rather than the focus happening early enough. I think this is good enough for now -- will ticket this improvement.